pandora-common/src/gameLogic/locks/lockLogic.ts

```diff
@@ -10,12 +10,19 @@ import type { AppearanceActionContext } from '../actionLogic/appearanceActions.t
 import type { LockDataBundle } from './lockData.ts';
 import type { LockSetup } from './lockSetup.ts';
 
+export const LockTimerOptionsSchema = z.object({
+	/** How long (in ms) the lock should be locked for, if it includes a timer. */
+	timer: z.number().int().nonnegative().optional(),
+	/** Whether the character that locked it is allowed to unlock the timer early, default true */
+	allowEarlyUnlock: z.boolean().optional(),
```

Comment:
Why make these properties optional at all, when the only thing done with the `timer` is missing is rejecting the request anyway? Making it non-optional removes need for checking that altogether and making `allowEarlyUnlock` non-optional simplifies its logic as well.